packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatusWheel/ProgressContent.tsx

```diff
@@ -0,0 +1,159 @@
+import {
+    selectCurrentCoinjoinWheelStates,
+    selectCurrentSessionDeadlineInfo,
+    selectRoundsDurationInHours,
+} from '@wallet-reducers/coinjoinReducer';
+import { useCoinjoinSessionBlockers } from '@suite/hooks/coinjoin/useCoinjoinSessionBlockers';
+
+export const Container = styled.div<{ isWide: boolean }>`
+    width: ${({ isWide }) => `calc(100% - ${isWide ? 12 : 8}px)`};
+    height: ${({ isWide }) => `calc(100% - ${isWide ? 12 : 8}px)`};
+    background: ${({ theme }) => theme.BG_WHITE};
+    border-radius: 50%;
+    transition: background 0.15s ease-out, width 0.15s ease-out, height 0.15s ease-out;
+`;
+
+const CenteringContainer = styled.div`
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    width: 80px;
+    height: 80px;
+    left: calc(50% - 40px);
+    top: calc(50% - 40px);
+`;
+const AllPrivateContent = styled.div`
+    padding-top: 2px;
+    color: ${({ theme }) => theme.TYPE_GREEN};
+
+    p {
+        font-size: ${variables.FONT_SIZE.H2};
+        line-height: 1;
+    }
+`;
+
+const StyledLoader = styled(FluidSpinner)`
+    opacity: 0.4;
+`;
+
+const TimeLeft = styled.p`
+    max-width: 80%;
+    color: ${({ theme }) => theme.TYPE_DARK_GREY};
+    font-size: ${variables.FONT_SIZE.H2};
+    line-height: 1;
+`;
+
+const iconBase = css`
+    margin-bottom: 4px;
+`;
+
+const PlayIcon = styled(Icon)`
+    ${iconBase}
+    margin-left: 4px;
+`;
+
+const PauseIcon = styled(Icon)`
+    ${iconBase};
+`;
+
+interface ProgressContentProps {
+    accountKey: string;
+    isWheelHovered: boolean;
+}
+
+export const ProgressContent = ({ accountKey, isWheelHovered }: ProgressContentProps) => {
+    const { isSessionActive, isPaused, isLoading, isAllPrivate, isAccountEmpty } = useSelector(
+        selectCurrentCoinjoinWheelStates,
+    );
+    const { sessionDeadline } = useSelector(selectCurrentSessionDeadlineInfo);
+    const roundsDurationInHours = useSelector(selectRoundsDurationInHours);
+
+    const theme = useTheme();
+    const { isCoinjoinSessionBlocked } = useCoinjoinSessionBlockers(accountKey);
+
+    const getProgressContent = () => {
+        const iconConfig = {
+            size: 25,
+            color: theme.TYPE_DARK_GREY,
+        };
+
+        const isLoadingIndicatorShown =
+            isLoading || (isSessionActive && !sessionDeadline && !isPaused);
+        const isPausedAndHovered = isPaused && isWheelHovered && !isCoinjoinSessionBlocked;
+        const isPausedOrRunningAndHovered = isPaused || (isSessionActive && isWheelHovered);
+
+        if (isAccountEmpty) {
+            return <PlayIcon icon="PLAY" {...iconConfig} />;
+        }
+
+        if (isLoadingIndicatorShown) {
+            return <StyledLoader size={40} strokeWidth={2} />;
+        }
+
+        if (isAllPrivate && !isSessionActive) {
+            return (
+                <AllPrivateContent>
+                    <p>100%</p>
```

Comment:
Not sure if FormattedNumber should be used here. Probably a nitpick. I guess some languages write it %100 and e.g. in czech there should be a non-breaking space 100 %, but not a big deal.

Reply:
Possible solution is to just make it translatable...

Reply:
I believe formatted number is better, essentially an auto translation :D

Reply:
but less flexible